Benzy-Clubbers/benzy-clubbers-ember
# Added Support for Ember-Bootstrap and Created Navbar

**What I Did:**
- Added support for Ember-Bootstrap
- Created starter for Navbar with basic items.

**How To Test:**
1. Run `npm install`.
2. Run `ember serve`.
3. You should see the Navbar.
